index.ts

```diff
@@ -229,47 +266,55 @@ export function bulk ({
  * @param {string} [$0.url]                     optional URL to override ESRI reverseGeocode endpoint
  * @return {Promise}                            A Promise that'll get resolved with reverse geocode result
  */
-export function reverse ({
+export function reverse({
   clientId,
   clientSecret,
   forStorage = false,
   point,
   url
 }: BaseQuery & {
-  forStorage?: boolean,
-  point: any
-}): Promise<BaseResponse> {
+  forStorage?: boolean
+  point: LonLatInput
+}): Promise<{ features: GeoJSON.FeatureCollection; query: Point }> {
```

Comment:
This is not the correct type since `GeoJSON.FeatureCollection` consists of an object with the `features` key.

Reply:
Ah thank you! Have added a comment about this as well in 8251d96fcd55c1a47faa4ece03b7a53c7122f955